requirements.txt

```diff
@@ -25,3 +25,6 @@ typing-extensions==3.7.4.3
 uvicorn==0.13.3
 wsproto==1.0.0
 zipp==3.4.0
+
+requests~=2.25.1
```

Comment:
requirements.txt is usually kept sorted by alphabetic order.

(and, we can eliminate python-dotenv)